lalrpop/src/main.rs

```diff
@@ -67,8 +75,10 @@ fn main1() -> io::Result<()> {
 const USAGE: &'static str = "
 Usage: lalrpop [options] <inputs>...
        lalrpop --help
+       lalrpop (-V | --version)
```

Comment:
I followed the same convention as rustc and cargo, Also it was suggested to be `-V` here https://github.com/nikomatsakis/lalrpop/issues/224#issue-222922059. 
But it can be lowercase, no matter to me.